src/util/updateName.ts

```diff
@@ -0,0 +1,5 @@
+import { User } from '../entity/User'
+
+export async function updateName(discordId, username: string): Promise<void> {
```

Comment:
Should we be calling this to "backfill" all the current bot users who don't have their username in the database? Maybe we can run this once per day for all users?